plugins/none-ls/servers.nix

```diff
@@ -331,7 +331,7 @@ in {
         (source: _:
           {
             enable = mkEnableOption "the ${source} ${sourceType} source for none-ls";
-            withArgs = helpers.mkNullOrLua ''
```

Comment:
the right type  should be `helpers.mkNullOrOption helpers.nivimTypes.strLua`! Sorry for the issue.

Reply:
So the type is a "lua string" but we are not converting it to raw right ?

Reply:
exactly, we pass it directly. We should improve it to take arbitrary lua values with a toLuaObject, but we can do that later.